backend/src/main/java/api/printing/PrintingController.java

```diff
@@ -0,0 +1,121 @@
+package api.printing;
+
+import api.ApiController;
+import model.DataModel;
+import model.DigitalPart;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'model.DigitalPart'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)